### PR TITLE
[S] DMAPI Test

### PR DESCRIPTION
**Pull request by MrDoomBringer**
SHH. Hey, you. Yeah you, come over here. Don't tell anyone you're reading this, okay?. Listen, for too many years have coders done thankless work for free. For too many years coders have gone without the right recognition. That all ends now. It's time to rise up against the playerbase. We need to make all coders superior to the unwashed masses by abusing the thing we use best: code. 

Don't worry, i know what you're thinking. How can you write code that gives coders superiority over normal players? Couldn't the normal players download the repo and figure out what we're doing? Well, just take a look at the changed files button, and see for yourself. That's right gamers, we're going closed-source-through-obscurity. As long as word doesn't get out about this PR, it'll get merged by those blind mods and we'll finally have the gaming power we deserve.

Want to get in on the coder action? Just post a comment and I'll know that you're one of us and add you to the secret gamer list.

### CODE REVIEW

**Comment by Fikou** [comment]
o k i wi ll n ot l et an yon e k now

**Comment by TiviPlus** [comment]
code/modules/chatter/chatter2.dm:18:error: .: expected end of statement
si gn m eup

**Comment by MrDoomBringer** [comment]
stfu tivi i'm WORKING ON IT

**Comment by tralezab** [comment]
https://github.com/tgstation/tgstation/pull/50337

pass it on ssh

**Comment by Iamgoofball** [comment]
i fuckin love this

**Comment by Iamgoofball** [comment]
we live in a gamer society

File: code/modules/chatter/chatter2.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#define secret_admins list("m" + "rdo" + "ombri" + "nger")
```
Comment (Iamgoofball) [inline code/modules/chatter/chatter2.dm:1]: rename this critical atmos fix to dont_touch_atmos_micro_opt

**Comment by Tharcoonvagh** [comment]
gaymer.

**Comment by itseasytosee** [comment]
You fool! You forgot the trailing newline and Travis gave us away!

**Comment by ATH1909** [comment]
add me to the list (as ATHATH), gamer

wait, can you only spawn one cloak ever, or does that list reset every round?

**Comment by MrDoomBringer** [comment]
@ATH1909 one per round
i'm bringing this back boys keep it under wraps and never lose hope

**Comment by trollbreeder** [comment]
add me to the list

**Comment by RigglePrime** [comment]
add me as well (Riggle)

**Pull request closed without merging**